mirage/header_layout.ml

```diff
@@ -0,0 +1,23 @@
+open Tyxml
+
+let header ?(page_title = "DNSVizor") () =
+  Html.(
+    head
+      (title (txt page_title))
+      [
+        meta ~a:[ a_charset "UTF-8" ] ();
+        meta
+          ~a:
+            [
+              a_name "viewport";
+              a_content "width=device-width, initial-scale=1.0";
+            ]
+          ();
+        script ~a:[ a_src "/main.js" ] (txt "");
+        link ~rel:[ `Stylesheet ] ~href:"/style.css" ();
```

Comment:
in this PR, we don't have the `main.js` neither the `style.css`. They should be added, maybe in a similar fashion than what is done on mollymawk (a directory `assets` that is `crunch`()ed into the binary)?